tests/foreman/ui_airgun/test_reporttemplates.py

```diff
@@ -146,6 +154,39 @@ def test_positive_unlock_report(session):
     """
 
 
+@tier2
+def test_positive_clone(session):
+    """Assure ability to clone a report template
+
+    :id: 912f1619-abb0-4e0f-88ce-88b5726fdbe0
+
+    :Steps:
+        1.  Go to Report template UI
+        2.  Choose a template and attempt to clone it
+
+    :expectedresults: The template is cloned
+
+    :CaseLevel: Integration
+    """
+    name = gen_string('alpha')
+    clone_name = gen_string('alpha')
+    content = gen_string('alpha')
+    with session:
+        session.reporttemplate.create({
+            'template.name': name,
+            'template.template_editor.editor': content,
+        })
+        assert session.reporttemplate.search(name)[0]['Name'] == name
+        session.reporttemplate.clone(
```

Comment:
Can you please add assertion on content as well? (equally important )

Reply:
This is a cloning test so I won't assert before cloning at all